packages/wallets/src/hooks/useSyncLocalStorageClientAccounts.ts

```diff
@@ -61,7 +61,7 @@ const useSyncLocalStorageClientAccounts = () => {
                     is_virtual: Number(data.is_virtual),
                     landing_company_name: newAccount.landing_company_shortcode,
                     landing_company_shortcode: newAccount.landing_company_shortcode,
-                    linked_to: [{ loginid: data.loginid, platform: 'dwallet' }],
+                    linked_to: [],
```

Comment:
why is this removed?

Reply:
Hi @amina-deriv as @matin-deriv mentioned in this comment https://github.com/binary-com/deriv-app/pull/14951#discussion_r1590488147 to remove "dwallet" as well. So I removed "dwallet" as well from then project.